pkgs/by-name/th/thcrap-steam-proton-wrapper/package.nix

```
@@ -0,0 +1,47 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  bash,
+  subversion,
+  gnome,
+}:
+with lib; let
+  version = "2b636c3";
+in
+  stdenv.mkDerivation {
+    pname = "thcrap-proton";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "tactikauan";
+      repo = "thcrap-steam-proton-wrapper";
+      rev = version;
+      sha256 = "sha256-J2O8F75NMdsxSaNVr8zLf+vLEJE6CHqWQIIscuuJZ3o=";
+    };
+
+    buildInputs = [bash subversion];
```

```suggestion
    buildInputs = [ subversion ];
```

`bash` should be in the standard environment by default